lib/formatters/basic_formatter.rb

```diff
@@ -89,6 +89,23 @@ def print()
       puts format_value(@msg_content_hashed ? StringUtils.sha1_hash(@message.body) : @message.body)
     end # print()
 
+    # Escapes characters which Python's eval() cannot load
+    # that is esp. \0, \r, \n. Use a range, to be safe.
+    def self.escape_chars(msg_string)
+      if msg_string.nil?
+        nil
+      else
+        msg_string.each_codepoint.map do |s|
+          # \\\\ because of python eval escaping
+          if s < 32 or s > 126
+            format('\\\\u%04x', s)
+          else
+            s.chr
+          end
+        end.join
```

Comment:
```suggestion
        end.join
```
messing around with feature